src/scripts/run-task.sh

```diff
@@ -89,4 +89,8 @@ set -- "$@" --task-definition "$ECS_PARAM_TASK_DEF"
 echo "Setting --cluster"
 set -- "$@" --cluster "$ECS_PARAM_CLUSTER_NAME"
 
-aws ecs run-task "$@"
+if [ -n "${ECS_PARAM_RUN_TASK_OUTPUT}" ]; then
+    aws ecs run-task "$@" >> "${ECS_PARAM_RUN_TASK_OUTPUT}"
```

Comment:
Simply redirecting the output will hide it from the UI output. We want the output in a file but we are not attempting to hide it from the UI. Use `tee` to do both.

<img width="388" alt="image" src="https://user-images.githubusercontent.com/33272306/173079327-195ecc4a-8743-4f83-bdf3-5c00686a4679.png">

Reply:
changed to `aws ecs run-task "$@" | tee "${ECS_PARAM_RUN_TASK_OUTPUT}"`